credible/faq.mdx

```diff
@@ -55,14 +49,14 @@ An assertion is a Solidity contract, that defines an invariant about your protoc
 Assertions are binary in nature, implemented through Solidity's **`require`** statement:
 
 - If the **`require`** condition is met, the transaction is valid
-- If the **`require`** fails, the transaction is invalid and will be filtered out
+- If the **`require`** fails, the transaction is invalid and will be dropped by the network
```

Comment:
nit: can you rewrite this to: "... is invalid and is dropped by the network"?

Fits better with the above sentence.